src/doc_scan_service/session/create/session.specification.js

```diff
@@ -47,6 +50,11 @@ class SessionSpecification {
     Validation.isInteger(clientSessionTokenTtl, 'clientSessionTokenTtl', true);
     this.clientSessionTokenTtl = clientSessionTokenTtl;
 
+    if (sessionDeadline) {
```

Comment:
Just wondering about the handling of `clientSessionTokenTtl` and `sessionDeadline` given they are mutually excluding, the API would reject, but I am considering maybe getting that handle by the SDK too, what do you think

Reply:
I was considering that too, but this comment says not handle on the SDK: https://lampkicking.atlassian.net/browse/SDK-1977?focusedCommentId=196347

Reply:
Ok, so from the comment, I understand that we would let the API handles the case when both are present, fine.
Then, I am a bit annoyed that the code is not 100% mirroring the handling of `clientSessionTokenTtl`, aka, that there is this check `if (sessionDeadline)`... but I appreciate that this is because the `Validation.instanceOf` does not have the `optional` parameter like `Validation.isInteger`. So, all good.